lib/matplotlib/colors.py

```diff
@@ -529,7 +529,18 @@ def __call__(self, X, alpha=None, bytes=False):
         if vtype == 'scalar':
             rgba = tuple(rgba[0, :])
         return rgba
-
+    
+    def __copy__(self):
+        """Create new object with the same class and update attributes
+	If object is initialized, copy the elements of _lut list
+	"""
+	cls = self.__class__
```

Comment:
My last commit (https://github.com/matplotlib/matplotlib/pull/8314/commits/3ece4cb11f6bfbb39b96e55509e647bb95f6f48a) has used lower level python to implement copy. The code has also used spaces instead of tabs